var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -100,6 +100,8 @@ class Hypre(AutotoolsPackage, CudaPackage, ROCmPackage):
         when="@2.12.1:",
     )
 
+    variant("force_rebuild", default=False, description="fixup to force rebuild")
```

Comment:
What does this variant do?

Reply:
exactly what it says, forces hypre to rebuild as a fixup :) removed now that I get the job with the logs, linked above